tools/ci-scripts/circleci-build-catalog-next.bash

```diff
@@ -9,13 +9,12 @@ sudo apt-get install solr-jetty libcommons-fileupload-java libpq-dev postgresql
 	 postgresql-contrib python-lxml postgresql-9.3-postgis-2.1 \
  	 python-dev libxml2-dev libxslt1-dev libgeos-c1 redis-server
 
-
 echo "-----------------------------------------------------------------"
 echo "Downliading settings"
 
 wget -O full_requirements.txt https://raw.githubusercontent.com/GSA/catalog.data.gov/master/ckan/requirements.txt
-wget https://raw.githubusercontent.com/GSA/ckan/datagov-newcatalog/test-core.ini
-wget https://raw.githubusercontent.com/GSA/ckan/datagov-newcatalog/ckan/config/who.ini
+wget https://raw.githubusercontent.com/ckan/ckan/ckan-2.8.6/test-core.ini
+wget https://raw.githubusercontent.com/ckan/ckan/ckan-2.8.6/ckan/config/who.ini
```

Comment:
Maybe stick the branch/tag in a variable.

Reply:
Did it [here](required to boto plugin to upload to S3 in some internal commands)